src/vcSettings.h

```diff
@@ -95,6 +97,21 @@ enum vcTileRendererFlags
   vcTRF_OnlyRequestVisibleTiles = 0x1,
 };
 
+enum vcImageFormats
+{
+  vcIF_PNG,
+  vcIF_BMP,
+  vcIF_TGA,
+  vcIF_JPG
+};
+
+enum vcImageResolutions
```

Comment:
This name is not suitable. `vcScreenshotOutputResolution` is better. There should also be a 'current scene window size' option